### PR TITLE
Add missing MutableSet abstract base class

### PR DESCRIPTION
**Pull request by dahlia**
[`MutableSet`][1] is a mutable version of [`Set`][2].

[1]: https://docs.python.org/3/library/collections.abc.html#collections.abc.MutableSet
[2]: https://docs.python.org/3/library/collections.abc.html#collections.abc.Set

### CODE REVIEW

File: stdlib/3/collections/__init__.pyi [hunk 1]
```diff
@@ -19,6 +19,7 @@ from typing import (
     MutableSequence as MutableSequence,
     MutableSet as MutableSet,
     AbstractSet as Set,
+    MutableSet as MutableSet,
```
Comment (gvanrossum) [inline stdlib/3/collections/__init__.pyi]: That's a duplicate from two lines above! :-)

Comment (dahlia) [inline stdlib/3/collections/__init__.pyi]: @gvanrossum Thanks for review.  Undone!

**Comment by gvanrossum** [comment]
Thanks! All set.

**Comment by dahlia** [comment]
Thank you!

**Pull request merged**